res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -19,30 +19,41 @@ limitations under the License.
     padding: 0 var(--cpd-space-3x);
     border-bottom: 1px solid $separator;
     background-color: $background;
+    transition: all 0.3s ease;
+    cursor: pointer;
+}
+
+.mx_RoomHeader:hover {
+    background-color: $header-panel-bg-hover;
 }
 
 .mx_RoomHeader_info {
-    cursor: pointer;
     flex: 1;
 }
 
+.mx_RoomHeader_truncated {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 1;
+
+    overflow: hidden;
+    word-break: break-all;
+    text-overflow: ellipsis;
+}
+
 .mx_RoomHeader_heading {
     display: flex;
     gap: var(--cpd-space-1x);
     align-items: center;
+
+    /* Spacing of 64px between the topic and the buttons, but we have
+       to account for the gap of the flex items */
+    padding-right: var(--cpd-space-13x);
```

Comment:
Something is a bit strange with a large gap growing at low widths

I don't see this gap in the Figma comps

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/e8a7c703-8288-4902-96a4-864cba8c9b7b)

Reply:
The topic fills the space fine

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/40baff69-0268-4760-aec5-4b018ef3989d)